### ISSUE TITLE
can i ask about reason of using adam optimization?

### ISSUE THREAD

**Issue by sooyeonshin** [epiception/theano-mc-cnn#3]
hi i'm studying about mc-cnn. 
so i'm very thankful about your work. (original lua based mc-cnn code is so complex)
I have one question about your code.
can i ask about reason of using adam optimization?
I can't find optimizer at original lua version code. but I guess it's not adam updater.
it's not affect train result? 
may be i misunderstand about original code and they originaly using adam optimization.

**Comment by epiception** [comment]
Hi,

Glad you liked it :smile: .
I only used Adam Optimizer in the code mainly to improve training speed and to ensure that I don't have to worry too much about learning rate decay and manual tuning. I guess the original paper mentions the use of the standard SGD method. Adam has become the preferred choice these days.
